### DISCUSSION TITLE
ScriptReference/Vector3.RotateTowards

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#701] [Notes]
# ScriptReference/Vector3.RotateTowards

https://docs.unity3d.com/ScriptReference/Vector3.RotateTowards.html

**Comment by ogike** [comment]
Might be obvious, but if the __current__ vector has a magnitude of 0, and maxMagnitudeDelta is 0, no rotation will occur.
